lib/yao/resources/restfully_accessible.rb

```diff
@@ -150,5 +156,10 @@ def return_resource(d)
     def return_resources(arr)
       arr.map{|d| return_resource(d) }
     end
+
+    def is_uuid?(str)
+      /^[\da-f]{32}$/ === str or
```

Comment:
there should be some hosts consist of 32 characters, but I have no idea about ignoring for that case...